data/test/scenarios/unknown_scenario_warning.cfg

```diff
@@ -95,12 +108,42 @@
 #enddef
 
 # The numbers in the names are (number of broken branches) (number of ok branches) (last scenario)
+#####
+# API(s) being tested: [endlevel]next_scenario
+##
+# Expected end state:
+# The next_scenario values are all identical, so log strict is broken with a warning referencing the singular next scenario.
+#####
 {TEST_UNKNOWN_SCENARIO "unknown_scenario_1_0"}
+#####
+# API(s) being tested: [endlevel]next_scenario
+##
+# Expected end state:
+# The next_scenario values are not all identical, so log strict is broken with a warning saying only some of the next scenarios weren't found.
+#####
 {TEST_UNKNOWN_SCENARIO "unknown_scenario_1_0_last" SCEN2=""}
+#####
+# API(s) being tested: [endlevel]next_scenario
+##
+# Expected end state:
+# The next_scenario values are not all identical, so log strict is broken with a warning saying only some of the next scenarios weren't found.
+#####
 {TEST_UNKNOWN_SCENARIO "unknown_scenario_1_1" SCEN2="test_return"}
+#####
+# API(s) being tested: [endlevel]next_scenario
+##
+# Expected end state:
+# The next_scenario values are all identical, so log strict is broken with a warning saying only some of the next scenarios weren't found and lists both scenarios which weren't found.
```

Comment:
There is more than one next_scenario value, but all of them are invalid, so log strict is broken with a warning saying all of the next scenarios weren't found and lists both scenarios which weren't found.

(The comment is wrong about all of the values being identical, one of them has an `_2` on the end.)